ci-operator/step-registry/mirror-images/check-registry-service/mirror-images-check-registry-service-commands.sh

```diff
@@ -26,7 +26,9 @@ function check_mirror_registry_response()
     while (( try < max_retries )); do
         echo "Checking mirror registry response code ${try}/${max_retries}"
         http_code=$(curl -o /dev/null -I -k -s -w "%{http_code}" "https://${BASTION_PUBLIC_ADDRESS}:5000")
-        if [[ "${http_code}" != "200" ]]; then
+        curl_ret=$?
+        if [[ "${curl_ret}" != "0" ]] || [[ "${http_code}" != "200" ]]; then
+            echo "curl exit code: ${curl_ret}"
```

Comment:
how about printing out the `http_code` together in the command?

Reply:
Updated